internal/helpers/prometheus_helper.go

```diff
@@ -156,11 +156,35 @@ func (mrs *MonitoringRuleSet) createAntecedentRule(metric, recordName, window st
 	}
 }
 
+// checks if the metric source type of the metric in the SLI is Prometheus-compatible
+func (mrs *MonitoringRuleSet) isPrometheusSource() bool {
+	sourceString := ""
+	opts := []string{mrs.Sli.Spec.RatioMetric.Total.MetricSource.Type, mrs.Sli.Spec.ThresholdMetric.MetricSource.Type}
+	for _, opt := range opts {
+		if opt != "" {
+			sourceString = opt
+			break
+		}
+	}
+
+	switch sourceString {
+	case
+		"Prometheus",
+		"Mimir",
+		"Cortex",
+		"VictoriaMetrics",
+		"Thanos",
+		"":
```

Comment:
This means that if the `MetricSource.Type` is empty, it will still try to create the PromehheusRule, won't it?

Reply:
Yeah, as we expect that by default the user of OSKO is using a Prometheus-compatible metricSource

Reply:
Still not fully convinced about this one.

Reply:
What would you change?

Reply:
Or rather, how would you change it

Reply:
Just leave the empty string out and handle the warning as it is incompatible with PromQ

Reply:
Done, we can always revert/change this one